WORKSPACE

```diff
@@ -97,6 +97,16 @@ load("@io_bazel_rules_sass//:defs.bzl", "sass_repositories")
 
 sass_repositories()
 
+http_archive(
```

Comment:
Let's add a note here about updating this, for future reference:

- We should ensure we always bump the requirements.txt `protobuf` dep to be `>=` the version here
- We probably should keep this in sync with TF, even if we're no longer using the org_tensorflow workspace to build, since there are other projects (like Keras) that do, and generating protobuf code at a more recent version than the `protobuf` runtime supplied by TF's bazel build tooling can lead to test failures for those projects.

Reply:
Thanks for the note!